datasets/at/pep/crawler.py

```diff
@@ -0,0 +1,68 @@
+from dateutil.parser import parse
+
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise
+
+
+def parse_german_date(date_string):
+    return parse(date_string, dayfirst=True, locale='de')
+
+def crawl_item(url_info_page: str, context: Context):
+
+    info_page = context.fetch_html(url_info_page)
+
+    first_name = info_page.xpath("//span[@itemprop='http://schema.org/givenName']/text()")[0]
+    last_name = info_page.xpath("//span[@itemprop='http://schema.org/familyName']/text()")[0]
+
+    person = context.make("Person")
+    person.id = context.make_id(first_name, last_name)
+
+    person.add("firstName", first_name)
+    person.add("lastName", last_name)
+
+    person.add("website", url_info_page)
+
+    try:
+        birth_date_in_german = info_page.xpath("//span[@itemprop='birthDate']/text()")[0]
+        person.add("birthDate", parse_german_date(birth_date_in_german))
+    except:
+        pass
```

Comment:
There's a matra "use exceptions for exceptional circumstances" - I think a better pattern and more consistent with our codebase would be

```suggestion
    birth_date_in_german = info_page.find("//span[@itemprop='birthDate']/text()")
    if birth_date_in_german:
        person.add("birthDate", parse_german_date(birth_date_in_german))
```

lxml el.find returns only the first match. You only need el.xpath when you want to do `contains()` because it's a bit more powerful, for some reason. Most of the time `find` and `findall` are sufficient.

Reply:
Nice, I didn't know about the `find` method.

Reply:
I got an error while trying to run the crawl because of the find method:

![Screenshot 2024-02-15 at 10 07 19](https://github.com/opensanctions/opensanctions/assets/37581842/3b47e41f-ce9e-4e03-b2af-1360d565a6fa)

Reply:
Hi @bgmello - sorry about the delayed response.
It looks like you need `el.find(".//tagname...`
i.e. a dot before the // to start at the current element.